versioned_docs/version-2023.1/03_server/10_integration/07_custom-endpoints/02_basics.md

```diff
@@ -2,200 +2,213 @@
 title: 'Custom endpoints - basics'
 sidebar_label: 'Basics'
 id: basics
-keywords: [server, integration, custom endpoints, basics]
+keywords: [ server, integration, custom endpoints, basics ]
```

Comment:
@kievitsp version 2023.1 didn't have the new endpoints I believe, it was version 6.5 back then so they are not available.